### PR TITLE
few constants added

### PR DESCRIPTION
**Pull request by btensay**
Few constants added. Plus, version number incremented.

https://tools.hmcts.net/jira/browse/ECM-406

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java [hunk 1]
```diff
@@ -339,21 +341,25 @@
     public static final String DUPLICATE_JURISDICTION_CODE_ERROR_MESSAGE = "Duplicate Jurisdiction Code(s) : ";
     public static final String MISSING_JURISDICTION_OUTCOME_ERROR_MESSAGE = "A Jurisdiction outcome is required before the case can be closed";
     public static final String MISSING_JURISDICTION_MESSAGE = "You cannot close this case as there is currently no jurisdiction and no jurisdiction outcome associated with it";
-    public static final String MISSING_JUDGEMENT_JURISDICTION_MESSAGE = "This case contains a Judgment which does not have a jurisdiction attached to it. To enable this case to be closed, please ensure the relevant jurisdiction(s) has been linked to the Judgment.";
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java:342]: Did @Angela-LW not add this is yesterday? Why is it being deleted

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java:342]: That was what I was asking.

Comment (Angela-LW) [reply src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java:342]: yes i added this for ECM-413, please do not delete

Comment (btensay) [reply src/main/java/uk/gov/hmcts/ecm/common/model/helper/Constants.java:342]: restored

**Review by gauravchawla73463 — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**